stdlib/2and3/logging/__init__.pyi

```diff
@@ -275,7 +275,7 @@ else:
     @overload
     def getLogger() -> Logger: ...
     @overload
-    def getLogger(name: str) -> Logger: ...
+    def getLogger(name: Union[unicode, str]) -> Logger: ...
```

Comment:
(Oh wait, I realize why -- this is in a Python 2 only branch. Still, it's better to use Text, so I changed that.) Will merge once tests pass again.